compiler/damlc/tests/daml-test-files/InterfaceRetroMissingMethod.daml

```diff
@@ -0,0 +1,30 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR range=15:11-15:12;Template InterfaceRetroMissingMethod:T is missing method m for interface InterfaceRetroMissingMethod:I
```

Comment:
This error should point to the retroactive implementation but it currently points to the template since we're reusing the same error constructor. We should either parametrize the constructor with an argument for the direction or equivalently duplicate and tweak the existing constructor. I'll do this in a separate PR